.github/workflows/daily_modin_precommit.yml

```diff
@@ -173,6 +173,68 @@ jobs:
             .tox/.coverage
             .tox/coverage.xml
 
+  test-pandas-patch-versions:
+    name: Test Snowpark pandas with pandas ${{ matrix.pandas-version }}
+    needs: build
+    runs-on: ${{ matrix.os.image_name }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - image_name: ubuntu-latest-64-cores
+            download_name: linux
+        pandas-version: ["2.2.1", "2.2.2"] # TODO include 2.2.3 once it's available in snowflake anaconda
```

Comment:
why do we need to wait for the snwoflake anaconda channel, that impacts stored sproc, notebook, ut shouldn't impact our github daily test

Reply:
but is 2.2.3 the one installed in our merge gate test?

Reply:
What's the merge gate test you're referring to? When I tried adding 2.2.3, all tests that used a UDF or sproc failed because it's not in anaconda (you can see an example in the daily job run where I forgot to pin the pandas version: https://github.com/snowflakedb/snowpark-python/actions/runs/11285715088/job/31389018772).

Reply:
For UDF and stored sproc it is expected, you can probably skip the udf and stored sproc test on pandas 2.2.3 for now, but for general tests, it should work